replica-common/src/main/java/com/graphhopper/stableid/PathDetailsBuilderFactoryWithStableId.java

```diff
@@ -69,7 +69,7 @@
             builders.add(new EdgeKeyDetails());
         }
 
-        if (requestedPathDetails.contains("r5_edge_id")) {
+        if (requestedPathDetails.contains("stable_edge_ids")) {
```

Comment:
what happens if we just remove this check?

Reply:
We absolutely could do that, and I think it would mean that we wouldn't need to add any query params to the GH requests in `RouteResource` and `PtRouteResource`, which is...even nicer 🤔 

Let me re-write that way, I like it!

Reply:
I like it, too. Interesting way of using an interesting Factory interface. :-)